src/Microsoft.DotNet.XHarness.CLI/CommandArguments/Apple/AppleTestCommandArguments.cs

```diff
@@ -107,13 +107,9 @@ public override void Validate()
                 throw new ArgumentException($"Failed to find the app bundle at {AppPackagePath}");
             }
 
-            if (RunTargets.Count == 0)
+            if (Target == null)
```

Comment:
Didn't you accidentally hide by overriding `Target` for `AppleTestCommandArguments`?

Reply:
No more `Target` anywhere anymore :)